src/lib/components/form/Form.component.tsx

```diff
@@ -22,7 +22,10 @@ import { Text } from '../text/Text.component';
 const DESCRIPTION_PREFIX = 'describe-';
 const maxWidthTooltip = { maxWidth: '20rem' };
 
-type FormProps = Omit<HTMLProps<HTMLFormElement>, 'ref' | 'as'> & {
+type FormProps = Omit<
+  HTMLProps<HTMLFormElement>,
+  'ref' | 'as' | 'noValidate' | 'formNoValidate'
```

Comment:
In practice `formNoValidate` is an input property not a form one. But ok for me this is really minor !

Reply:
Yes, I saw that, but the IDE was suggesting it on the form element for some reason, so I've made the decision to remove it in order to not mislead people in the future